exercises/concept/port-palermo/port_palermo.rb

```diff
@@ -1,6 +1,4 @@
 module Port
-  # TODO: define the 'Identifier' constant
```

Comment:
The TODO could remain, with the updated information.  Otherwise, if this is a "no-brainer" line of code, why is it simply not part of the delivered material?